integrations/disco/hooks/use-disco-issue-credential.ts

```diff
@@ -0,0 +1,98 @@
+import { setTimeout } from 'timers'
+
```

Comment:
```suggestion
```

Reply:
as handleToast is not being used,I'll remove the entire things